Sorting/bubble.cpp

```diff
@@ -16,9 +17,7 @@ void Sort::BubbleSort(){
 
     f(i, sorted.size() - 1)
         f(j, sorted.size() - 1)
-            if(sorted[j] > sorted[j + 1]){
-                int temp = sorted[j];
-                sorted[j] = sorted[j+1];
-                sorted[j+1] = temp; 
-            }
+            if(sorted[j] > sorted[j + 1])
+                std::swap(sorted[j], sorted[j + 1]);
+
```

Comment:
This is a type-2 change. Please make a separate PR for it, and revert it from this one. One PR must have changes of a single type only. Please read README.md for more information.